com.unity.shadergraph/Editor/GraphDeltaRegistry/Generation/Interpreter.cs

```diff
@@ -31,6 +31,11 @@ void GetBlock(ShaderContainer container, CustomizationPoint vertexCP, Customizat
             return builder.ToString();
         }
 
+		public static string GetShaderForGraph(IGraphHandler graph, Registry.Registry registry)
+        {
+            return null;
```

Comment:
Is this a stub for something that is going to come later?

Reply:
There seem to be a couple stubs in this code. That's fine but I would like to see that called out, since you'll know what you're stubbing and others won't.

Reply:
good catch! this one in particular is never used and I don't quite recall what the purpose is, so I can remove it safely 👍 
Will do a comb of the files to see if there are any other stubs that need commenting

Reply:
Sg